src/pages/Pools/PairsByToken0/index.tsx

```diff
@@ -0,0 +1,201 @@
+  const router = useRouter()
+  const token0 = useToken(currencyIdA)
+  const { loading: loadingPairs, pairs } = usePairsByToken0(token0)
+  const liquidityMiningEnabled = useLiquidityMiningFeatureFlag()
+
+  const [openTokenModal, setOpenTokenModal] = useState(false)
+
+  const handleAllClick = useCallback(() => {
+    setOpenTokenModal(true)
+  }, [])
+
+  const handleModalClose = useCallback(() => {
+    setOpenTokenModal(false)
+  }, [])
+
+  const handleCurrencySelect = useCallback(
+    token => {
+      router.push({
+        pathname: `/pools/${token.address}`
+      })
+    },
+    [router]
+  )
+
+  if (token0 === undefined) {
+    return <Redirect to="/pools" />
+  }
+  return (
+    <>
+      <PageWrapper>
+        <SwapPoolTabs active={'pool'} />
+
+        <AutoColumn gap="lg">
+          <AutoColumn gap="lg" style={{ width: '100%' }}>
+            <TitleRow style={{ marginTop: '1rem' }} padding={'0'}>
+              <Flex alignItems="center">
+                <Box mr="8px">
+                  <UndecoratedLink to="/pools">
+                    <TYPE.mediumHeader fontWeight="400" fontSize="26px" lineHeight="32px" color="text4">
+                      Pairs
+                    </TYPE.mediumHeader>
+                  </UndecoratedLink>
+                </Box>
+                <Box mr="8px">
+                  <TYPE.mediumHeader fontWeight="400" fontSize="26px" lineHeight="32px" color="text4">
+                    /
+                  </TYPE.mediumHeader>
+                </Box>
+                <PointableFlex onClick={handleAllClick}>
+                  <Box mr="4px">
+                    <CurrencyLogo currency={token0 || undefined} size="20px" />
+                  </Box>
+                  <Box mr="4px">
+                    <Text fontWeight="600" fontSize="16px" lineHeight="20px">
+                      {token0?.symbol}
+                    </Text>
+                  </Box>
+                </PointableFlex>
+                <Box>
+                  <ChevronDown size={12} />
+                </Box>
+              </Flex>
+              <ButtonRow>
+                <ResponsiveButtonPrimary id="join-pool-button" as={Link} padding="8px 14px" to="/create">
+                  <Text fontWeight={700} fontSize={12}>
+                    CREATE PAIR
+                  </Text>
+                </ResponsiveButtonPrimary>
+                {liquidityMiningEnabled && (
+                  <ResponsiveButtonSecondary as={Link} padding="8px 14px" to="/liquidity-mining/create">
+                    <Text fontWeight={700} fontSize={12} lineHeight="15px">
+                      CREATE LIQ. MINING
+                    </Text>
+                  </ResponsiveButtonSecondary>
+                )}
+              </ButtonRow>
+            </TitleRow>
+            <PairsList loading={loadingPairs} pairs={pairs} />
+          </AutoColumn>
+        </AutoColumn>
+        {/* TODO: uncomment once multi-chain analytics are a thing */}
```

Comment:
I heard it's a thing now :)

Reply:
Right, links reenabled here fb17a2be8f62ea406cb251719ea161d666b891c6 (only if an account is actually connected).